chapters/synchronous.tex

```diff
@@ -883,6 +883,13 @@ \subsection{Connected Components of the Equations and Variables Graph}\label{con
 \emph{Connected components}) of this graph with appropriate definition of
 the incidence operator.
 
+A special case is the built-in variable \lstinline!time! (see \cref{built-in-variable-time}).
+Each use of \lstinline!time! is conceptually included as a separate variable in this analysis, $\mathit{time}_i$ with $\text{\lstinline!der!}(\mathit{time}_{i}) = 1$.
```

Comment:
Is it obvious that this is a trivial change?  Couldn't `time` have been defined using `interval()` instead to avoid the need for discretization?  Just asking.

Reply:
There are always alternatives.

However, as I understand it: forbidding direct use of time for the non-discretized variant was part of the design goal (since expressions such as `time>=0.2` have issues with numerical noise that we want to avoid), and you can see that examples use `sample(time)`; and clearly that is one option - but that clashes too much with having every continuous model in a sampled systems. Thus the above keeps the previous intentions.

Reply:
I suppose it comes down to whether you want the use of `time` to imply the need for time discretization and a `solverMethod`, or if you want to be able to use `time` in a clocked discrete-time partition?

Reply:
Yes. And it is clear that the original design was to not allow `time `directly in clocked discrete-time partitions (to offer stronger guarantees for synchronous), and this keeps that design - while still allowing `time `with `solverMethod`, since it is not worse than a differential equation.

Reply:
@bernhard-thiele @MartinOtter Do you have any input on this?

Reply:
11 days. I guess not